src/Tribe/Linked_Posts.php

```diff
@@ -708,12 +758,14 @@ public function handle_submission_by_post_type( $event_id, $linked_post_type, $s
 		}
 
 		$temp_submission = $submission;
-		$submission = array();
+		$submission      = array();
 
 		// make sure all elements are arrays
 		foreach ( $temp_submission as $key => $value ) {
 			$submission[ $key ] = is_array( $value ) ? $value : array( $value );
 		}
+		// Cleanup
+		unset( $temp_submission, $key, $value );
```

Comment:
I'm wondering about the reason behind this `unset` calls?

Reply:
just to be explicit about them not being used in following lines of code. as the comment says, `Cleanup` is all...

Reply:
Okay I don't think that might be necessary as as soon execution is outside this method variables are no longer available.

Reply:
You're correct. It's not necessary, but I thought it helpful since it's a large method. I can remove it if you think it best to do so. Please confirm...

Reply:
Yeah I think for this one might have sense as you can confuse the variable after is no longer "valid".

So I think is a good call 👍

Reply:
Removed both `unset`s